### PR TITLE
SALEOR-2007 Add import sorter

### PR DESCRIPTION
**Pull request by piotrgrundas**
Add `eslint-plugin-simple-import-sort`

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Changes are mentioned in the changelog.

### Test Environment Config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -75,7 +80,33 @@
...
+          [
+            "^(@components|@paths|@pages|@utils|@hooks|@types|@app|@locale|@temp)(/.*|$)"
+          ],
+          // Relative imports.
+          // Anything that starts with a dot.
+          ["^\\."],
+          // Assets.
+          ["^(images|!!raw-loader)"],
+          // Styles
+          ["w*.*css|^@styles"]
```
Comment (jwm0) [inline .eslintrc:106]: The `@styles` helper is mostly for style utilities, I suggest bundling it above with other absolute imports (aliases)

**Review by jwm0 — COMMENTED** [review]
Looks good! 🎉

**Comment by krzysztofwolski** [comment]
Can we merge it after nextjs merges with master? Would like to avoid situation, where all 256 files have shuffled imports and commit message is "add next js" (we are squashing PRs)

**Comment by piotrgrundas** [comment]
@krzysztofwolski That's the plan :D

**Comment by piotrgrundas** [comment]
@jwm0 U mentioned u created some ticket for this? Could you link it with the ticket? 🙏

**Comment by patrys** [comment]
Task linked: [SALEOR-2007 Streamline eslint config across frontend projects](https://app.clickup.com/t/2549495/SALEOR-2007)

**Review by jwm0 — APPROVED** [review]

**Pull request merged**